script.js

```diff
@@ -12,60 +12,169 @@ const confirmationButton = document.getElementById("confirmation-button");
 const cartItemsContainer = document.getElementById("cart-items-container");
 const cartItemsNumber = document.getElementById("cart-items-number");
 const cartButton = document.getElementById("cart-button");
+const emptyCartModal = document.getElementById("empty-cart-modal");
+const cartModal = document.getElementById("cart-modal");
+const cartSummary = document.getElementById("cart-summary");
 
-let cartItems = 0;
+let cartItems = [];
+let selectedPack = {
+  name: "",
+  price: 0,
+  quantity: 0,
+  icon: "",
+};
+let cartClicked = false;
+
+const closeModal = () => {
+  confirmationModalOverlay.classList.add("opacity-0", "pointer-events-none");
+  confirmationModalOverlay.classList.remove("opacity-100");
```

Comment:
suggestion:
`confirmationModalOverlay.classList.replace("opacity-100", "opacity-0");`
`confirmationModalOverlay.classList.add("pointer-events-none");`

to improve readability and consistency with the next line